pages/web-literacy/weblitdata.js

```diff
@@ -0,0 +1,33 @@
+var categories = require('./categories');
```

Comment:
do we still use this file, or is it a left-over from the initial circletree testing?